layouts/shortcodes/observability_pipelines/destination_settings/syslog.md

```diff
@@ -1,4 +1,19 @@
-The following fields are optional:
+<div class="alert alert-warning">The Rsyslog and Syslog-ng destinations support the <a href="https://datatracker.ietf.org/doc/html/rfc5424">RFC5424</a> format and are only available when you use the Rsyslog or Syslog-ng source. </div>
+
+The Rsyslog and Syslog-ng destinations match these log fields to the following Syslog fields:
+
+| Log Event       | SYSLOG FIELD | Default                    |
+|-----------------|--------------|----------------------------|
+| log["message"]  | MESSAGE      | `NIL`                      |
+| log["procid"]   | PROCID       | `<OPW PID>    `            |
+| log["appname"]  | APP-NAME     | `observability_pipelines`  |
+| log["facility"] | FACILITY     | `8 (log_user)`             |
+| log["msgid"]    | MSGID        | `NIL`                      |
+| log["severity"] | SEVERITY     | `info`                     |
+| log["host"]     | HOSTNAME     | `NIL`                      |
+| log["timestamp"]| TIMESTAMP    | `Utc::now()`               |
```

Comment:
For the default- We could probably say "Current UTC time"

Reply:
updated in this [commit](https://github.com/DataDog/documentation/pull/23623/commits/670b6a9e3ca69e26c5e6d4362a5a91e92bc94aec)